challenge.js

```diff
@@ -43,6 +49,8 @@
 
 // ANSWER
 
+  // document.querySelector("p");
+
```

Comment:
True! This selector will retrieve that `<p>`, though it will also retrieve any other `<p>`s that might exist.